reactive/filebeat.py

```diff
@@ -190,6 +191,92 @@ def check_filebeat_repo():
     remove_state('filebeat.repo.changed')
 
 
+@when('apt.installed.filebeat')
+@when_any('config.changed.ignore_older', 'config.changed.clean_inactive',
+          'config.changed.scan_frequency')
+def sanity_check():
+    ignore_older = str(config().get('ignore_older'))
+    clean_inactive = str(config().get('clean_inactive'))
+    scan_frequency = str(config().get('scan_frequency'))
+
+    log("Executing sanity checks on: ignore_older = {}, \
+        clean_inactive = {}".format(ignore_older, clean_inactive))
+
+    # sanity check config strings 
+    if check_string(ignore_older) and check_string(clean_inactive):
+        log("Charm config sanity checks passed. Restarting Filebeat")
+        service("restart", "filebeat") # required?
+    else:
+        msg = "Invalid config. Sanity check(s) marked as False failed: \
+            ignore_older = {}, clean_inactive = {}".format(
```

Comment:
this is a bit obscure, would it be perhaps better to have separate log lines if the validation of either string fails? A small explanation of what "invalid" means would also be nice

Reply:
done